node_modules/oae-content/lib/api.js

```diff
@@ -1294,7 +1294,13 @@ var removeContentFromLibrary = module.exports.removeContentFromLibrary = functio
                         return callback(err);
                     }
 
-                    LibraryAPI.Index.remove(ContentConstants.library.CONTENT_LIBRARY_INDEX_NAME, [ libraryOwnerId ], content.id, content.lastModified, function(err) {
+                    var entry = {
```

Comment:
Actually there are different variations of the same entity. E.g., the difference between an entry that is provided for insert (`rank`) and an entry that is provided for update (`oldRank`/`newRank`). So it's probably best we just keep it as a simple object with specific properties based on the library index method.